tests/integration/data/test-bundle-etcd.yaml

```
@@ -0,0 +1,30 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: integration-test-etcd
+description: |-
+  Used to deploy or refresh within an integration test model
+series: jammy
```

can we test with focal?
```suggestion
series: focal
```